core/src/executor/execute.rs

```diff
@@ -375,16 +374,54 @@ pub async fn execute<T: GStore + GStoreMut>(
 
             Ok((storage, Payload::ShowIndexes(indexes)))
         }
-        //- Metadata
-        #[cfg(feature = "metadata")]
         Statement::ShowVariable(variable) => match variable {
-            Variable::Tables => storage
-                .schema_names()
-                .await
-                .map(|table_names| Payload::ShowVariable(PayloadVariable::Tables(table_names)))
-                .try_self(storage),
+            Variable::Tables => {
+                let query = Query {
+                    body: SetExpr::Select(Box::new(crate::ast::Select {
+                        projection: vec![SelectItem::Expr {
+                            expr: Expr::Identifier("TABLE_NAME".to_string()),
+                            label: "TABLE_NAME".to_string(),
+                        }],
+                        from: TableWithJoins {
+                            relation: TableFactor::Dictionary {
+                                dict: Dictionary::GlueTables,
+                                alias: TableAlias {
+                                    name: "GLUE_TABLES".to_owned(),
+                                    columns: Vec::new(),
+                                },
+                            },
+                            joins: Vec::new(),
+                        },
+                        selection: None,
+                        group_by: Vec::new(),
+                        having: None,
+                    })),
+                    order_by: Vec::new(),
+                    limit: None,
+                    offset: None,
+                };
+
+                let rows = try_block!(storage, {
+                    let rows = select(&storage, &query, None).await?;
+                    let rows = rows
+                        .map_ok(|Row(values)| values)
+                        .try_collect::<Vec<_>>()
+                        .await?;
+                    Ok(rows)
+                });
+
+                let table_names = rows
+                    .iter()
+                    .flat_map(|values| values.iter().map(|value| value.into()))
+                    .collect::<Vec<_>>();
+
+                Ok((
+                    storage,
+                    Payload::ShowVariable(PayloadVariable::Tables(table_names)),
+                ))
+            }
             Variable::Version => {
-                let version = storage.version();
+                let version = env!("CARGO_PKG_VERSION").to_owned();
```

Comment:
```suggestion
                let version = std::env::var("CARGO_PKG_VERSION")
                    .unwrap_or(env!("CARGO_PKG_VERSION").to_owned());
```

Reply:
`env!` is a macro, so only the version of `gluesql-core` is specified.
We want to follow the `Cargo.toml` version of the storage library.